stetho-timber/src/main/AndroidManifest.xml

```diff
@@ -7,9 +7,13 @@
   ~ of patent rights can be found in the PATENTS file in the same directory.
   -->
 
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-          package="com.facebook.stetho.timber">
+<manifest
+    xmlns:tools="http://schemas.android.com/tools"
+    package="com.facebook.stetho.timber">
 
-    <application />
+    //fixes Timber Library min sdk issue
+    <uses-sdk tools:overrideLibrary="timber.log"/>
```

Comment:
Hi, you are correct, but these changes cause failure in travis build, and I try to fix it but I couldn't.

Sent from my iPhone

> On May 10, 2016, at 2:10 AM, Josh Guilfoyle <notifications@github.com> wrote:
> 
> In stetho-timber/src/main/AndroidManifest.xml:
> 
> >  
> > -    <application />
> > +    //fixes Timber Library min sdk issue
> > +    <uses-sdk tools:overrideLibrary="timber.log"/>
> It seems like this entry is the only thing that would actually fix your issue. Why are there so many unrelated changes (like bumping our build tools version)?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>